fMRISurface/GenericfMRISurfaceProcessingPipeline.sh

```diff
@@ -139,25 +160,39 @@ T1wFolder="$Path"/"$Subject"/"$T1wFolder"
 ResultsFolder="$AtlasSpaceFolder"/"$ResultsFolder"/"$NameOffMRI"
 ROIFolder="$AtlasSpaceFolder"/"$ROIFolder"
 
-#Make fMRI Ribbon
-#Noisy Voxel Outlier Exclusion
-#Ribbon-based Volume to Surface mapping and resampling to standard surface
-
-log_Msg "Make fMRI Ribbon"
-log_Msg "mkdir -p ${ResultsFolder}/RibbonVolumeToSurfaceMapping"
-mkdir -p "$ResultsFolder"/RibbonVolumeToSurfaceMapping
-"$PipelineScripts"/RibbonVolumeToSurfaceMapping.sh "$ResultsFolder"/RibbonVolumeToSurfaceMapping "$ResultsFolder"/"$NameOffMRI" "$Subject" "$AtlasSpaceFolder"/"$DownSampleFolder" "$LowResMesh" "$AtlasSpaceFolder"/"$NativeFolder" "${RegName}"
-
-#Surface Smoothing
-log_Msg "Surface Smoothing"
-"$PipelineScripts"/SurfaceSmoothing.sh "$ResultsFolder"/"$NameOffMRI" "$Subject" "$AtlasSpaceFolder"/"$DownSampleFolder" "$LowResMesh" "$SmoothingFWHM"
+# ------------------------------------------------------------------------------
+#  Start work
+# ------------------------------------------------------------------------------
 
-#Subcortical Processing
-log_Msg "Subcortical Processing"
-"$PipelineScripts"/SubcorticalProcessing.sh "$AtlasSpaceFolder" "$ROIFolder" "$FinalfMRIResolution" "$ResultsFolder" "$NameOffMRI" "$SmoothingFWHM" "$GrayordinatesResolution"
+if ((doProcessing)); then
+    #Make fMRI Ribbon
+    #Noisy Voxel Outlier Exclusion
+    #Ribbon-based Volume to Surface mapping and resampling to standard surface
+	log_Msg "Make fMRI Ribbon"
```

Comment:
This appears to contain tabs, while most lines use spaces for indenting, please change to spaces.

Reply:
There are also sections of the file that use 2 spaces for indenting, while elsewhere we mostly use 4, it would be nice if that was made consistent at 4.

Reply:
I'm not working on anything else, so please feel free to fix the tab/spacing issue.